cmake/Modules/FindNCCL.cmake

```diff
@@ -14,32 +14,34 @@
 # install NCCL in the same location as the CUDA toolkit.
 # See https://github.com/caffe2/caffe2/issues/1601
 
-set(NCCL_ROOT_DIR $ENV{NCCL_ROOT_DIR} CACHE PATH "Folder contains NVIDIA NCCL")
+set(NCCL_INCLUDE_DIR $ENV{NCCL_INCLUDE_DIR} CACHE PATH "Folder contains NVIDIA NCCL headers")
+set(NCCL_LIB_DIR $ENV{NCCL_LIB_DIR} CACHE PATH "Folder contains NVIDIA NCCL libraries")
+set(NCCL_VERSION $ENV{NCCL_VERSION} CACHE STRING "Version of NCCL to build with")
+
+list(APPEND NCCL_ROOT ${NCCL_ROOT_DIR} ${CUDA_TOOLKIT_ROOT_DIR})
```

Comment:
I think you probably meant `$ENV{NCCL_ROOT_DIR}` here? `NCCL_ROOT_DIR` is never set otherwise.

Reply:
Yes, nice catch. I changed this to ENV and added a deprecation message. It used to be set in cmake.py but somehow I lost it during rebase.